common.py

```diff
@@ -5,16 +5,16 @@
 import csv
 from datetime import datetime
 import re
-from typing import Iterable, Union
+from typing import Iterable, Union, Dict
 
 
 # Commonly used dimensions
 COURTYARD_LINE_WIDTH = 0.1
 
 
-def init_cache(uuid_cache_file: str) -> collections.OrderedDict:
+def init_cache(uuid_cache_file: str) -> Dict[str, str]:
```

Comment:
We can't annotate it as `OrderedDict` if we want to support Python 3.5, see https://stackoverflow.com/questions/41207128/how-do-i-specify-ordereddict-k-v-types-for-mypy-type-annotation#43583996